docs/v1/use-cases.md

```diff
@@ -65,6 +75,66 @@ whether deleting a broker should:
 2. Leave orphaned Service Instances in the Service Catalog
 3. Fail if Service Instances still exist for the broker
 
+
+### Search and Browsing Services
+
+#### Searching Services
+
+Consumers should be be able to search or filter their catalog by labels. For
+example, if I search for all services with 'catalog=database' the catalog
+will return the list of services that match that label. This assumes, of
+course, that producers are able to label their service offerings.
+
+#### Service Metadata
+
+Each service should have a list of metadata that it exposes in the catalog.
+If we're following the Cloud Foundry model you can view the list of metadata
+fields [here](https://docs.cloudfoundry.org/services/catalog-metadata.html).
+
+We should consider what metadata needs to be exposed for a strong CLI and UI
+experience. Here are some suggestions for metadata fields:
+
+    * name
+    * short description
+    * long desciption
+    * documentation/support urls
+    * icon URL
+    * image URLs - a list of images that could be displayed in a UI
+    * TOS (terms of service) link
+    * a list of plans
+        * plan name
+        * plan description
+        * plan cost
+    * construction parameters
```

Comment:
Should we be talking about bindings at all here? Binding parameters are also needed.